packages/core/integration-tests/test/svg-react-typescript.js

```diff
@@ -0,0 +1,32 @@
+import assert from 'assert';
+import {bundle, outputFS} from '@parcel/test-utils';
+import path from 'path';
+
+describe('svg-react-typescript', () => {
```

Comment:
Please add this test to https://github.com/parcel-bundler/parcel/blob/58823e43b8717cf499e92667da380a9b942a4afe/packages/core/integration-tests/test/svg-react.js#L5 instead (and as one test with all assertions, similar to the existing one)

Reply:
@mischnic all done, thanks for the feedback